Calculator/Calculator/Model/Enumeration/ExpressionParser.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  ExpressionParser.swift
+//  Calculator
+//
+//  Created by jiye Yi on 2023/01/27.
+//
+
+enum ExpressionParser {
+    static func parse(from input: String) -> Formula {
+        let components = componentsByOperator(from: input)
+        var operands = CalculatorItemQueue<Double>()
+        var operators = CalculatorItemQueue<Operator>()
+        
+        components.compactMap({ Double($0) }).forEach{ operands.enqueue($0) }
+        components.filter{ $0.count == 1 }.compactMap({ Operator(rawValue: Character($0)) }).forEach{ operators.enqueue($0) }
```

Comment:
고차함수의 활용 아주 좋습니다 👍👍

가독성을 위해 꼭 아래와 같은 방법은 아니더라도 어느 정도 개행을 해주시면 이해하기 편할 것 같네요!
```suggestion
        components.compactMap { Double($0) }
            .forEach{ operands.enqueue($0) }
        
        components.filter { $0.count == 1 }
            .compactMap { Operator(rawValue: Character($0)) }
            .forEach{ operators.enqueue($0) }
```